app/views/partners/show.html.erb

```diff
@@ -46,9 +46,8 @@
         <% if @partner.has_service_areas? %>
           <p>We operate in <%= partner_service_area_text(@partner) %>.</p>
         <% end %>
-        <%= render_component "address",
-          address: @partner.address
-        %>
+
+        <%= render AddressComponent.new(address: @partner.address) %>
```

Comment:
@lenikadali i found one more address just fyi! otherwise happy to merge if you are?

Reply:
Hahaha beat me to it :smile: 

Yes please merge :raised_hands:

Reply:
oh no theres an test fail now are you ok to fix the test?

Reply:
Not sure why that is. Locally it doesn't fail though. Let me see :crossed_fingers: